### ISSUE TITLE
How to generate Figure 4 ?

### ISSUE THREAD

**Issue by scutfrank** [cvqluu/Additive-Margin-Softmax-Loss-Pytorch#17]
Hi,I wonder how to generate Figure 4 Row Two in the paper CosFace: Large Margin Cosine Loss for Deep Face Recognition.Would you mind to share the code?Thanks!
